proj/src/main/webapp/script.js

```diff
@@ -117,15 +117,15 @@ function appendMessage(payload){
  */
 
 const CHAT_ID = '-MB0ycAOM8VGIXlev5u8'
-const PATH = '/chat/'+CHAT_ID+'/messages'; // can make this more detailed (for example add user ID)
+const PATH = getPath("test", CHAT_ID);
```

Comment:
Since you have a function to return path, I think you can remove this constant now.